nf_core/pipeline-template/{{cookiecutter.name_noslash}}/conf/charliecloud.config

```diff
@@ -0,0 +1,22 @@
+/*
+ * -------------------------------------------------
+ *  Nextflow config file for Charliecloud
+ * -------------------------------------------------
+ * Assumes that pipeline dependencies are all available in
+ * /opt/conda/envs/{{ cookiecutter.name_noslash }}-{{ cookiecutter.version }}/bin
+ * If multiple containers are used, it may be necessary to include their environment
+ * paths in the `env` scope below
+ */
+
+charliecloud {
+  enabled = true
+}
+
+manifest {
+  nextflowVersion = '>=20.12.0-edge'
+}
+
+env {
+  PATH = "/opt/conda/bin:/opt/conda/envs/{{ cookiecutter.name_noslash }}-{{ cookiecutter.version }}/bin:$PATH"
+  // TODO nf-core: If the pipeline uses additional environments, add them to $PATH as well
```

Comment:
Any idea how this will work with BioContainers as used in DSL2 pipelines?

Reply:
Should work as the path will always be the same for Biocontainers: 

https://github.com/nf-core/tools/issues/824#issuecomment-757865592